INSTALL

```diff
@@ -479,8 +479,7 @@
   enable-tls1_3
                    TODO(TLS1.3): Make this enabled by default
                    Build support for TLS1.3. Note: This is a WIP feature and
-                   does not currently interoperate with other TLS1.3
-                   implementations! Use with caution!!
+                   may have nasty bugs.  Use with caution!!
```

Comment:
My understanding is that this text dates from when "openssl TLS 1.3" was mostly just regular TLS 1.2 with some numbers changed.
We currently are (basically) in compliance with draft-19, and should interoperate with other draft-19 implementations.  Because of the per-draft versions, we should not negotiate "TLS 1.3" with a peer speaking draft-18 and die on the protocol differences; we will instead just negotiate 1.2 and communicate.  So, I am not quite convinced that leaving the old text is correct.
(But, of course, we also have better things to do than argue over it, like fixing other TODO(TLS1.3) entries, so I won't press any further.)